homeassistant/components/spokestack/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Spokestack TTS component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["tts"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Hello World component."""
+    # Ensure our name space for storing objects is a known type. A dict is
+    # common/preferred as it allows a separate instance of your class for each
+    # instance that has been created in the UI.
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
```

Comment:
Not needed
```suggestion
```

Reply:
I ran into this issue: `homeassistant.component.tts` doesn't have the attribute 'async_setup_entry'. Does that mean that I do need `async_setup` for the tts platform?